raiden/blockchain/events.py

```diff
@@ -92,10 +92,13 @@ def get_contract_events(
         if filter_ is not None:
             filter_.uninstall()
 
-    return [
-        translator.decode_event(event['topics'], event['data'])
-        for event in events
-    ]
+    result = []
```

Comment:
Didn't see this issue before. Already implemented it (as stated above), but to apply it to all other events, asked for some thoughts [here](https://github.com/raiden-network/raiden/issues/450#issuecomment-318934759).